### PR TITLE
API now loads to challenges.html site 

### PR DESCRIPTION
**Pull request by Tobias-hubs**
Fixed API that now loads to challenges.html via challenges.js script. 
CSS for grid (very basic at the moment) 
One existion red button takes user to challenges.html. (there will be more buttons with same functionality, and some "temporary buttons needs to be removed" )

### CODE REVIEW

**Comment by jennymakki** [comment]
Good job! Maybe the naming of elements should follow BEM?

**Comment by marcusmoller97** [comment]
I think we need to change how you add the data from data.challenges into the card. 
What i mean is this code:
         challengeItem.innerHTML = `
           <h2>${challenge.title}</h2>
           <p>${challenge.description}</p>
           ${challenge.title}
           <p>Rating: ${challenge.rating}</p>
           <p>Type: ${challenge.type}</p>
           <p>Participants: ${challenge.minParticipants} - ${challenge.maxParticipants}</p>
         `;
We need to add it in a way so we can use the the divs and style that Jörgen already have design since before. With the current method we cant add the style.

Maybe use APPEND and create the divs according to the style in Jörgens hacker escape room. Then you can append the h2, p, img and the others according to each div with the classes that Jörgen have made.

Otherwise, great job!

**Comment by Tobias-hubs** [comment]
Thanks for feedback,  im fixing that

File: index.html [hunk 1]
```diff
@@ -342,6 +340,18 @@ <h3 class="card__title">Title of room (on-site)</h3>
         >
       </div>
     </section>
+<!--    Buttons for All Challenges Tobias -->
```
Comment (Klumpish) [inline index.html:343]: i think we can remove this button?

**Review by Klumpish — COMMENTED** [review]
great job it looks nice, some minor things that need to be removed to clean it up a bit. good job

**Pull request merged**